_posts/2025-04-04-business-processes-with-camunda-tools--open-mic-announcement.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: "Upcoming Open Mic - Business processes with Camunda tools"
+categories: [ Open Mic Announcement, Open Mic ]
+---
+
+The next Open mic session starts on Friday 11th April 2025 at 9:30
+at [this link](https://meet.jit.si/open-mic-kbss). [Martin Ledvinka](https://kbss.felk.cvut.cz/web/team#martin-ledvinka)
+will talk about his experience with using tools from Camunda to model and execute organization business processes.
+
+{% include figure image_path="/assets/images/posts/2025-04-04-openmic-workflow.png" alt="OpenMic assignment and
+announcement process" %}
+
+##### Abstract
+
+Most organizations operate based on _business processes_. Information systems are developed to support these processes,
+often implicitly.
+However, more complex processes require and explicit approach - modeling the process using BPMN and using a _workflow
```

Comment:
Require an explicit? What is BPMN?

Reply:
The explicit approach is described just after the hyphen - modeling the process and using a workflow engine to run it.
I assume most people with some IT background have heard about BPMN (business process model notation)